docs/CQ.rst

```diff
@@ -0,0 +1,59 @@
+Create a Contribution Questionnaire
+===================================
+
+All the new dependencies and all the dependencies updates should be submited as a new CQ into the
+`ECLIPSE IP Zilla <https://dev.eclipse.org/ipzilla/query.cgi>`__.
+
+Submitting a CQ
+---------------
+
+Visit `https://projects.eclipse.org/projects/locationtech.geotrellis/ <https://projects.eclipse.org/projects/locationtech.geotrellis/>`__
+and log into the system.
+
+.. figure:: img/locationtech-geotrellis.png
+   :alt: The GeoTrellis project page
+
+On the right, in the ``COMMITTER TOOLS`` side bar, you can find a link `Create a Contribution Questionnaire <https://projects.eclipse.org/projects/locationtech.geotrellis/cq/create>`__
+
+.. figure:: img/geotrellis-cq.png
+   :alt: Create a Contribution Questionnaire page
+
+Next, we need to create a Third-Party Code Request usually. All the next steps would be based on a particular library example.
+Let's create a CQ for ``pureconfig v0.10.2``:
+
+.. figure:: img/pb-cq.png
+   :alt: A "piggyback" CQ
+
+If your library of your version is here, then select it. It is the easiest result and it means that somebody have already sent a request to
+the ``Eclipse IP`` team and the dependency (probably) had been validated. It means that this dep can be approved automatically.
+
+Let's create a CQ for ``pureconfig v0.11.0``. There is no such a library, so we'll have to fill the page manually:
+
+.. figure:: img/npb-cq-intro.png
+   :alt: A new CQ
+
+The next step would be to full fill the information about it:
+
+.. figure:: img/npb-cq.png
+   :alt: A new CQ Step 1
+
+The question is - how to do it? If this library but of a previous version was already submitted, we can use it as an example to fill
```

Comment:
"If the library was already submitted but a new version is being requested, one can look at what was already submitted for that library to use as an example. In our case we can use `Pureconfig Version 0.10.2`"